cli/src/proxy.ts

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios'
```

Comment:
Yes, I did not realized we have test for `cli`.  Now added test for `cli proxies.` 
I don't know for what reason, but all of the test for cli is not active since they set as a `it.skip` and most of them are outdates. 

For this issue I will create make a issue and will work on them later :)